api-client/src/Endpoint.ts

```diff
@@ -83,6 +84,7 @@ class Endpoint<P, T> {
       const fetchedData = this.mapResponse(json, params)
       return new Payload(false, url, fetchedData, null)
     } catch (e) {
+      console.error(e)
```

Comment:
Is it best practice to log the error? Just asking :)

Reply:
Not sure but I definitely find it useful. It is stripped in production anyway.